### PR TITLE
Re-enable the Dart Development Service (DDS)

### PR DESCRIPTION
**Pull request by bkonyi**
This change re-enables DDS and outputs the DDS URI in place of the VM service URI on the console. If --disable-dds is not provided, --host-vmservice-port will be used to determine the port for DDS rather than the host port for the VM service, which will instead be randomly chosen.

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/dds.dart [hunk 1]
```diff
@@ -17,13 +17,17 @@ class DartDevelopmentService {
 
   // TODO(bkonyi): enable once VM service can handle SSE forwarding for
```
Comment (bkonyi) [inline packages/flutter_tools/lib/src/base/dds.dart:18]: Self-nit: remove this flag / TODO

**Comment by jonahwilliams** [comment]
what remains to be done after this PR? Just web support?

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by bkonyi** [comment]
> what remains to be done after this PR? Just web support?

Yup, that should be it. DDS has SSE support now, so it's just a matter of wiring things up hopefully.

**Pull request merged**